torch/nn/parallel/distributed.py

```diff
@@ -194,23 +194,23 @@ class DistributedDataParallel(Module):
         >>>     dist_optim.step()
 
     .. warning::
-        Using DistributedDataParallel in conjuction with the
+        Using DistributedDataParallel in conjunction with the
         :ref:`distributed-rpc-framework` is experimental and subject to change.
 
     Args:
         module (Module): module to be parallelized
         device_ids (list of int or torch.device): CUDA devices. This should
                    only be provided when the input module resides on a single
-                   CUDA device. For single-device modules, the ``i``th
+                   CUDA device. For single-device modules, the ``i``\ th 
```

Comment:
Fixes weird formatting in https://pytorch.org/docs/master/generated/torch.nn.parallel.DistributedDataParallel.html#distributeddataparallel for the ** device_ids** parameter